waspc/src/Wasp/Analyzer/Parser/Ctx.hs

```diff
@@ -23,15 +25,18 @@ withCtx f (WithCtx ctx expr) = f ctx expr
 -- usually refer to a region and not just one position/char in the code.
 -- This is captured in an issue https://github.com/wasp-lang/wasp/issues/404 .
```

Comment:
Can you now remove this TODO comment?

Reply:
Arg, my source selection got messed up. Was trying to highlight this area:
```
-- | Gives parsing context to AST nodes -> e.g. source position from which they originated.
-- TODO: Instead of having just SourcePosition, it would be better to have SourceRegion, since errors
-- usually refer to a region and not just one position/char in the code.
-- This is captured in an issue https://github.com/wasp-lang/wasp/issues/404 .
data Ctx = Ctx
  { ctxSourceRegion :: SourceRegion
  }
  deriving (Show, Eq)
  ```

Reply:
Oh yes thanks, I completely missed it! And I put it here above the place where I knew I will be making the change from pos into region, so I see it, and I still missed it :D.